### PR TITLE
Cinq next dns hijack tracking graphql

### PR DESCRIPTION
**Pull request by riot-jetaylor**

### CODE REVIEW

File: cloud-inquisitor/graph/server.go.bak [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package main
```
Comment (HTTP500) [inline cloud-inquisitor/graph/server.go.bak]: do we want to exclude this file?

File: cloud-inquisitor/resource.go [hunk 2]
```diff
@@ -72,18 +57,35 @@ type Resource interface {
 // Specifically for Auditing resource which can have a tag
 type TaggableResource interface {
```
Comment (HTTP500) [inline cloud-inquisitor/resource.go:58]: what type do we think of S3?

File: cloud-inquisitor/secrets/vault/vault.go [hunk 3]
```diff
@@ -0,0 +1,119 @@
+package vault
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/RiotGames/cloud-inquisitor/cloud-inquisitor/settings"
+	"go.riotgames.com/OpSec/rVault"
```
Comment (HTTP500) [inline cloud-inquisitor/secrets/vault/vault.go:10]: What's OpSec?

File: cloud-inquisitor/graph/server.go.bak (hunk 1 above)
Comment (riot-jetaylor) [reply cloud-inquisitor/graph/server.go.bak]: yes, will remove.

File: cloud-inquisitor/resource.go (hunk 2 above)
Comment (riot-jetaylor) [reply cloud-inquisitor/resource.go:58]: S3 can be both Taggable and Hijackable.

File: README.md [hunk 4]
```diff
@@ -271,6 +321,63 @@ terraform {
...
+
+    name = "cinq-next"
+    environment = "dev"
+}
+```
+
+### Vault
+
 
 ## Terraform Deploy
```
Comment (HTTP500) [inline README.md:384]: missing something the line below?

File: README.md [hunk 5]
```diff
@@ -271,6 +321,63 @@ terraform {
...
+```hcl
+module "project_role" {
+    source = "./terraform_modules/project_role"
+
+    name = "cinq-next"
+    environment = "dev"
+}
+```
+
+### Vault
```
Comment (HTTP500) [inline README.md]: do we planned to add vault intro later?

Comment (riot-jetaylor) [reply README.md]: added the necessary comments/description of how to use vault

File: README.md (hunk 4 above)
Comment (riot-jetaylor) [reply README.md:384]: yes, also added in deployment steps

**Review by HTTP500 — APPROVED** [review]

**Pull request merged**